budget-manager/src/modals/add-spending/add-spending-dialog.html

```diff
@@ -0,0 +1,29 @@
+<form>
+    <h1 mat-dialog-title>Add Spending</h1>
+    <div mat-dialog-content [formGroup]="form">
+        <mat-form-field>
+            <mat-label>Value</mat-label>
+            <input type="number" matInput formControlName="value" />
+        </mat-form-field>
+        <mat-form-field>
+            <mat-label>Name</mat-label>
+            <input type="text" matInput formControlName="name" />
+        </mat-form-field>
+        <mat-form-field appearance="fill">
+            <mat-label>Category</mat-label>
+            <mat-select formControlName="category">
+                <mat-option *ngFor="let category of data" value="{{category.name}}">{{category.name}}</mat-option>
```

Comment:
use bracket syntax for inputs
```suggestion
                <mat-option *ngFor="let category of data" [value]="category.name">{{category.name}}</mat-option>
```

Reply:
fixed